pkg/kube/exec.go

```diff
@@ -80,14 +111,16 @@ func ExecWithOptions(kubeCli kubernetes.Interface, options ExecOptions) (string,
 		TTY:       tty,
 	}, scheme.ParameterCodec)
 
-	config, err := LoadConfig()
-	if err != nil {
-		return "", "", err
-	}
-
-	var stdout, stderr bytes.Buffer
-	err = execute("POST", req.URL(), config, options.Stdin, &stdout, &stderr, tty)
-	return strings.TrimSpace(stdout.String()), strings.TrimSpace(stderr.String()), err
+	pro, pwo := io.Pipe()
+	pre, pwe := io.Pipe()
+	errCh := make(chan error, 1)
```

Comment:
What is the purpose of `errCh` here? It is not referenced out of the goroutine below.